File: src/main/java/io/github/support/terminal/application/domains/core/api/slack/models/Message.java
package io.github.support.terminal.application.domains.core.api.slack.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class Message {

    @JsonProperty("type")
    private String type;

    @JsonProperty("user")
    private String user;

    @JsonProperty("text")
    private String text;

    @JsonProperty("ts")
    private Double ts;

}

File: src/main/kotlin/io/github/bot/terminal/application/domains/integrations/telegram/models/Message.kt
package io.github.bot.terminal.application.domains.integrations.telegram.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

import java.util.*

@JsonIgnoreProperties(ignoreUnknown = true)
data class Message(
        @JsonProperty("message_id") val messageId: String? = null,
        @JsonProperty("from") val from: User? = null,
        @JsonProperty("chat") val chat: Chat? = null,
        @JsonProperty("date") val date: Date? = null,
        @JsonProperty("text") val text: String = ""
)